haas/cli.py

```diff
@@ -27,6 +27,8 @@
 
 command_dict = {}
 usage_dict = {}
+MIN_PORT_NUMBER = 0
```

Comment:
Ah, had another look. You're using strictly less than in the check so this works actually, but in my mind "MIN" implies an inclusive range, so the name is a bit confusing.